apps/prairielearn/src/pages/shared/syncHelpers.js

```diff
@@ -1,123 +1,29 @@
 // @ts-check
+
 const { ECR, ECRClient } = require('@aws-sdk/client-ecr');
 const _ = require('lodash');
 const ERR = require('async-stacktrace');
-const fs = require('fs-extra');
 const async = require('async');
 const Docker = require('dockerode');
 const { DockerName, setupDockerAuth } = require('@prairielearn/docker-utils');
-const namedLocks = require('@prairielearn/named-locks');
 
 const { makeAwsClientConfig } = require('../../lib/aws');
 const { config } = require('../../lib/config');
 const { createServerJob } = require('../../lib/server-jobs');
-const syncFromDisk = require('../../sync/syncFromDisk');
-const courseUtil = require('../../lib/courseUtil');
+const course = require('../../lib/course');
 const util = require('util');
-const chunks = require('../../lib/chunks');
 const debug = require('debug')('prairielearn:syncHelpers');
-const { getLockNameForCoursePath } = require('../../lib/course');
 
 const docker = new Docker();
 
 module.exports.pullAndUpdate = async function (locals) {
-  const serverJob = await createServerJob({
+  const { jobSequenceId } = await course.pullAndUpdate({
```

Comment:
Nit: typically we've been naming functions so that they can be used via named imports (in this case, `pullAndUpdateCourse`).

Reply:
Done in b2d20abdc.